source/run_lcao.cpp

```diff
@@ -114,11 +114,10 @@ void Run_lcao::lcao_line(void)
 		Exx_Abfs::Lmax,
 		GlobalC::ucell.infoNL.nprojmax,
 		GlobalC::ucell.infoNL.nproj,
-		GlobalC::ucell.infoNL.Beta);
+        GlobalC::ucell.infoNL.Beta);
```

Comment:
In this file, why not move line "ORB_control orb_con;" from line 24 to line 86?

Reply:
OK, I've moved it.